src/components/frame/v5/pages/VerifiedPage/partials/TableItem/TableItem.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC } from 'react';
+
+import { TableItemProps } from './types';
+import { useColonyContext, useUserReputation } from '~hooks';
+import UserAvatarPopover from '~v5/shared/UserAvatarPopover';
+import { splitWalletAddress } from '~utils/splitWalletAddress';
+import Icon from '~shared/Icon';
+import { calculatePercentageReputation } from '~utils/reputation';
+import styles from './TableItem.module.css';
+import Checkbox from '~v5/common/Checkbox';
+
+const TableItem: FC<TableItemProps> = ({ member, onDeleteClick, onChange }) => {
+  const { colony } = useColonyContext();
+  const { colonyAddress } = colony || {};
+
+  const { user } = member || {};
+  const { walletAddress, name, profile } = user || {};
+  const { bio } = profile || {};
+  const { userReputation, totalReputation } = useUserReputation(
+    colonyAddress,
+    walletAddress,
+  );
+  const reputationPercentage = calculatePercentageReputation(
+    userReputation,
+    totalReputation,
+  );
+
+  return (
+    <div className={styles.tableItem}>
+      <div className="flex items-center">
+        <Checkbox
+          id={`verified-${name}`}
+          name={`verified-${name}`}
+          onChange={(e) => onChange(e)}
+        />
+        <div className="ml-1 flex">
+          <UserAvatarPopover
+            userName={name}
+            walletAddress={splitWalletAddress(walletAddress || '')}
+            aboutDescription={bio || ''}
+            // @TODO: add colonyReputationItems
+            // colonyReputation={colonyReputationItems}
+            user={user}
+            avatarSize="xs"
+            isVerified
+            // permissions={permissionsItems}
+          />
+        </div>
+        <span className="ml-1 flex shrink-0 text-blue-400">
+          <Icon name="verified" appearance={{ size: 'tiny' }} />
+        </span>
+      </div>
+      <div className="hidden sm:flex items-center">status</div>
+      <div className="hidden sm:flex items-center">
+        <Icon name="star" appearance={{ size: 'small' }} />
+        <span className="ml-1 text-sm text-gray-600">
+          {reputationPercentage || '0'}%
+        </span>
+      </div>
+      <div className="hidden sm:flex items-center">permissions</div>
```

Comment:
add missing translaction

Reply:
This is going to be permissions icons, so I left string as placeholder